types/index.d.ts

```diff
@@ -2681,6 +2691,7 @@ export interface ElementCommands {
    */
   click(
     selector: Definition,
+    force: boolean,
```

Comment:
The `force` option should be part of the new Element API commands.

Reply:
The `click` with `force` param is only available on `ScopedElement` and not on Selenium's `WebElement`.
This should be documented properly in the docs where we mention that when you await you get Selenium's `WebElement` and when used without await you get `ScopedElement`.